go/apps/api/openapi/openapi.yaml

```
@@ -3306,50 +3306,19 @@ components:
-                  description: |
-                    Lists permissions to evaluate against the key's assigned permissions and roles.
-                    Each permission supports hierarchical matching with wildcard notation.
-                    Empty arrays are not allowed - omit the permissions field entirely for no authorization.
-                  example: ["documents.read", "documents.write", "users.view"]
-              required:
-                - type
-                - permissions
-              additionalProperties: false
+          type: string
+          minLength: 1
+          maxLength: 1000 # Allow for complex permission queries
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)